src/musical/internal/domain/artist_service.go

```diff
@@ -32,3 +32,38 @@ func (service *ArtistService) GetArtist(ctx context.Context, id uuid.UUID) (mode
 	}
 	return artist, nil
 }
+
+func (service *ArtistService) AddArtist(ctx context.Context, artist models.Artist) error {
```

Comment:
В других сервисах по-другому называем.
```suggestion
func (service *ArtistService) CreateArtist(ctx context.Context, artist models.Artist) error {
```